db.py

```diff
@@ -0,0 +1,32 @@
+import records
+import os
+from config import config
+
+class DB:
+    def __init__(self):
+        params = self.read_config()
+        self.postgres_db = records.Database(f"postgresql://{params['user']}:{params['password']}@{params['host']}:{params['port']}/{params['database']}")
+
+    def read_config(self):
+        params = {}
+        params['user'] = os.environ.get('TTT_DB_USER', None)
+        if params['user']:
+            params['password'] = os.environ.get('TTT_DB_PASSWORD', None)
+            params['host'] = os.environ.get('TTT_DB_HOST', None)
+            params['port'] = os.environ.get('TTT_DB_PORT', None)
+            params['database'] = os.environ.get('TTT_DB_NAME', None)
+        else:
+            params = config()
+
+        return params
+
+    def get_best_move_from_db(self, computer_marker, board):
+        board_spots = ''.join(map(str, board.spots))
+        query = "SELECT best_move FROM computer_best_moves WHERE computer_marker=:computer_marker AND board_spots=:board_spots"
+        move = self.postgres_db.query(query, computer_marker=computer_marker, board_spots=board_spots)
+        return move
+
+    def insert_best_move_into_db(self, computer_marker, board, best_move):
+        board_spots = ''.join(map(str, board.spots))
```

Comment:
```py
board_spots = ''.join(map(str, board.spots))
```
Is duplicated on line 30 and line 24. Can you move this logic to a more appropriate class?

Reply:
resolved in https://github.com/olgaboiar/ttt_python/pull/8/commits/3eb32790195d129740900d935714a1938a88b035
- duplication removed
- a method to turn board spots array into string created in Board class